src/algebra/ordered_monoid.lean

```diff
@@ -49,10 +49,38 @@ class ordered_add_comm_monoid (α : Type*) extends add_comm_monoid α, partial_o
 
 attribute [to_additive] ordered_comm_monoid
 
+/-- A linearly ordered additive commutative monoid. -/
+@[protect_proj, ancestor ordered_add_comm_monoid linear_order]
```

Comment:
These ancestor attributes are in the wrong order, they should match the order in extends